addons/mail/static/src/services/new_message_service/new_message_service.js

```diff
@@ -0,0 +1,27 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+
+const { Component } = owl;
+
+export const newMessageService = {
+    dependencies: ["command"],
+    start(newEnv, { command }) {
+        command.add("Message a user", () => {
+            const env = Component.env;
+            const messagingMenu = env.messaging && env.messaging.messagingMenu;
+            if (!env.messaging.device.isMobile) {
+                env.messaging.chatWindowManager.openNewMessage();
+                messagingMenu.close();
+            } else {
+                messagingMenu.toggleMobileNewMessage();
+            }
```

Comment:
Did you actually test this in mobile? The shortcut doesn't work at all, and opening it from ctrl+k only works if the messaging menu is already opened. Also it has the side effect of closing it if it's already open.
I think for mobile it should do both open menu + display the mobile search (and not with toggles).

Reply:
I just checked with Nathan, shortcuts are not accessible from mobile. I'll remove the logic related to this.